src/api/audit.ts

```diff
@@ -0,0 +1,50 @@
+import { D2ApiGeneric, Id, Ref } from "..";
+import { D2ApiResponse } from "./common";
+import { PaginatedObjects, Pager } from ".";
+
+// https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/audit.html
+
+export class Audit {
+    constructor(public d2Api: D2ApiGeneric) {}
+
+    /* Return paginated audits for data values */
+    getDataValues(
+        options: DataValuesAuditOptions
+    ): D2ApiResponse<PaginatedObjects<DataValueAudit>> {
+        return this.d2Api
+            .get<GetDataValueAuditResponse>("/audits/dataValue", { ...options, paging: true })
+            .map(({ data }) => ({ pager: data.pager, objects: data.dataValueAudits }));
+    }
+}
+
+export type AuditType = "UPDATE" | "DELETE";
+
+// Apparently, the official docs are wrong (they reference a skipPaging property).
+// Standard properties paging/page/pageSize work.
+export type DataValuesAuditOptions = {
+    ds?: Id[];
+    de?: Id[];
+    pe?: string[];
+    ou?: Id[];
+    auditType?: AuditType;
+    // paging: boolean;
```

Comment:
@tokland Is it normal that this property is commented?

Reply:
```suggestion
```

Just tested, you force paging in line 15

Reply:
> @tokland Is it normal that this property is commented?

Typical "just know that this property exist", but can be removed